src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -1855,8 +1855,24 @@ public boolean addRemote(String siteId, String remoteName, String remoteUrl, Str
             } else {
                 params.put("remotePassword", remotePassword);
             }
-            params.put("remoteToken", remoteToken);
-            params.put("remotePrivateKey", remotePrivateKey);
+            if (StringUtils.isNotEmpty(remoteToken)) {
+                logger.debug("Encrypt token before inserting to database");
+                TextEncryptor encryptor = new PbkAesTextEncryptor(studioConfiguration.getProperty(SECURITY_CIPHER_KEY),
```

Comment:
Please move the encryptor declaration to the top of the method and use it for all encryption